packages/devextreme/js/viz/series/points/bar_point.js

```diff
@@ -46,9 +46,16 @@ export default _extend({}, symbolPoint, {
     },
 
     _calculateVisibility: function(x, y, width, height) {
+        this.inVisibleArea = this._getPointVisibility(x, y, width, height, true);
+    },
+
+    _getPointVisibility: function(x, y, width, height, allowEqual) {
```

Comment:
I suggest u use object parameter to make last boolean prop pass more explicit

Reply:
ok